service-runner/src/main/java/org/stellar/anchor/platform/ServiceRunner.java

```diff
@@ -49,7 +56,7 @@ public static void main(String[] args) {
       }
 
       if (cmd.hasOption("kotlin-reference-server") || cmd.hasOption("all")) {
-        startKotlinReferenceServer(null, true);
+        startKotlinReferenceServer(new HashMap<>(), true);
```

Comment:
This change is probably out of scope of this PR

Reply:
I will revert it. It was added because `envMap` parameter of `startKotlinReferenceServer` is not nullable and I couldn't start Kotlin Reference Server

Reply:
Hmm I'm pretty sure it should be nullable

Reply:
Yes, i get following exception:
```
Exception in thread "main" java.lang.NullPointerException: Parameter specified as non-null is null: method org.stellar.reference.RefenreceServerStartKt.start, parameter envMap
        at org.stellar.reference.RefenreceServerStartKt.start(RefenreceServerStart.kt)
        at org.stellar.anchor.platform.ServiceRunner.startKotlinReferenceServer(ServiceRunner.java:104)
        at org.stellar.anchor.platform.ServiceRunner.main(ServiceRunner.java:58)
```

Reply:
Got it I will fix